analyzer/src/funTest/assets/projects/synthetic/conan-expected-output-py.yml

```diff
@@ -0,0 +1,94 @@
+---
+project:
+  id: "Conan::Poco:1.9.3"
+  definition_file_path: "analyzer/src/funTest/assets/projects/synthetic/conan-py/conanfile.py"
+  declared_licenses:
+  - "The Boost Software License 1.0"
+  declared_licenses_processed:
+    unmapped:
+    - "The Boost Software License 1.0"
+  vcs:
+    type: "Git"
+    url: "http://github.com/pocoproject/conan-poco"
+    revision: ""
+    path: ""
+  vcs_processed:
+    type: "Git"
+    url: "<REPLACE_URL>"
+    revision: "<REPLACE_REVISION>"
+    path: "<REPLACE_PATH>"
+  homepage_url: ""
+  scopes:
+  - name: "build_requires"
+    dependencies: []
+  - name: "requires"
+    dependencies:
+    - id: "Conan::OpenSSL:1.0.2o"
+      dependencies:
+      - id: "Conan::zlib:1.2.11"
+    - id: "Conan::zlib:1.2.11"
+packages:
+- package:
+    id: "Conan::OpenSSL:1.0.2o"
+    purl: "pkg:conan/OpenSSL@1.0.2o"
+    declared_licenses:
+    - "The current OpenSSL licence is an 'Apache style' license: https://www.openssl.org/source/license.html"
+    declared_licenses_processed:
+      unmapped:
+      - "The current OpenSSL licence is an 'Apache style' license: https://www.openssl.org/source/license.html"
+    description: "OpenSSL is an open source project that provides a robust, commercial-grade,\
+      \ and full-featured toolkit for the Transport Layer Security (TLS) and Secure\
+      \ Sockets Layer (SSL) protocols"
+    homepage_url: ""
+    binary_artifact:
+      url: ""
+      hash:
+        value: ""
+        algorithm: ""
+    source_artifact:
+      url: ""
+      hash:
+        value: ""
+        algorithm: ""
+    vcs:
+      type: "Git"
+      url: "http://github.com/lasote/conan-openssl"
+      revision: "0"
```

Comment:
This "0" is something that Conan provides by default. Since keeping this "0" does not make much sense, we could add a check that if revision is "0" then we just put there empty string. Do you think that this would be better approach?

Reply:
Yes, I think that would be better. "0" is not a valid Git revision after all, and for consistency with other package managers we should use `""` to indicate "no revision" rather than using `"0"`.